.github/workflows/coveralls.yml

```diff
@@ -106,7 +106,7 @@ jobs:
         cat /proc/cpuinfo
         ./util/opensslwrap.sh version -c
     - name: make test
-      run: make test TESTS='-test_external_krb5'
+      run: make test TESTS='-test_external_krb5' EVP_TEST_EXTENDED=1
```

Comment:
Probably not part of this PR, but I think we need the concept of partial versus full testing for all tests, not just evp_test.

Reply:
Agreed, let's get this in first and fix the nomenclature later.
This PR is addressing coverage testing and this change does exactly that.